tests/hpc/mpi_master.pm

```diff
@@ -18,23 +18,26 @@ use warnings;
 use testapi;
 use lockapi;
 use utils;
-use registration qw(add_suseconnect_product get_addon_fullname);
 use version_utils 'is_sle';
 
 sub run {
     my $self          = shift;
     my $version       = get_required_var("VERSION");
     my $arch          = get_required_var("ARCH");
+    my $sdk_version   = get_required_var("BUILD_SDK");
     my $nodes         = get_required_var("CLUSTER_NODES");
     my $mpi           = get_required_var("MPI");
     my $mpi_c         = "simple_mpi.c";
     my @cluster_nodes = $self->cluster_names();
     my $cluster_nodes = join(',', @cluster_nodes);
     $version =~ s/-SP/./;
+    $sdk_version = "Build" . $sdk_version;
 
-    add_suseconnect_product(get_addon_fullname('sdk'), $version, $arch);
-    ## as the test code requires compilation, it should be possible to install required compilers
+    ## adding required sdk and gcc
+    ##TODO: consider better ways of handling addons/modules etc
+    ## https://progress.opensuse.org/issues/54773
     if (is_sle '<15') {
+        zypper_call("ar -f ftp://openqa.suse.de/SLE-12-SP5-SDK-POOL-$arch-$sdk_version-Media1/ SDK");
```

Comment:
```suggestion
        zypper_call("ar -f ftp://openqa.suse.de/SLE-$version-SDK-POOL-$arch-Build$sdk_version-Media1/ SDK");
```

Reply:
Anyway, I wonder if http://download.suse.de/ibs/SUSE/Products/SLE-SDK/12-SP5/x86_64/ is also the same, or you need specific build?